soc/xtensa/intel_adsp/cavs/include/intel_cnl_adsp/adsp_memory.h

```diff
@@ -25,6 +25,8 @@
 /* Host shared memory windows */
 #define HP_SRAM_WIN0_BASE (L2_SRAM_BASE + CONFIG_ADSP_WIN0_OFFSET)
 #define HP_SRAM_WIN0_SIZE 0x2000
+#define HP_SRAM_WIN2_BASE (HP_SRAM_WIN0_BASE + 0x4000)
```

Comment:
Any reason to define this base address differently than the others?

Reply:
Ack, will address this. Also noting conflict with #49077 and discussion at https://github.com/zephyrproject-rtos/zephyr/pull/49077/files#r950450740